spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'user/index', type: :system do
+  before(:each) do
+    driven_by(:rack_test)
+    @user = User.first
+  end
+
+  it 'has username' do
+    visit users_path
+    expect(page).to have_content(@user.name)
+  end
+
+  it 'has profile picture' do
+    visit users_path
+    expect(page).to have_css('img')
+  end
+
+  it 'has number of posts' do
+    visit users_path
+    expect(page).to have_content("Number of Posts: #{@user.posts_count}")
+  end
```

Comment:
- [ ] Please add a test that redirects to the user's show page when you click on a user as required in [Here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=When%20I%20click%20on%20a%20user%2C%20I%20am%20redirected%20to%20that%20user%27s%20show%20page.).